README.md

```diff
@@ -387,3 +387,15 @@ To enable logging in UMF source files please follow the guide in the
 ## Notices
 
 The contents of this repository may have been developed with support from one or more Intel-operated generative artificial intelligence solutions.
+
+## Anthem
```

Comment:
It should be on top of the file. also IMHO we need preamble not anthem

Reply:
It could be a part of the `Introduction` section

Reply:
Fixed.